pyproject.toml

```diff
@@ -1,3 +1,69 @@
+[tool.poetry]
+name = "rdflib"
+version = "6.3.0a0"
+description = """
+RDFLib is a Python library for working with RDF, \
+a simple yet powerful language for representing information."""
+authors = ["Daniel 'eikeon' Krech <eikeon@eikeon.com>"]
+maintainers = ["RDFLib Team <rdflib-dev@googlegroups.com>"]
+repository = "https://github.com/RDFLib/rdflib"
+documentation = "https://rdflib.readthedocs.org/"
+license = "BSD-3-Clause"
+classifiers=[
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "License :: OSI Approved :: BSD License",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Operating System :: OS Independent",
+    "Natural Language :: English"
+]
+readme = "README.md"
+
+[tool.poetry.scripts]
+rdfpipe = 'rdflib.tools.rdfpipe:main'
+csv2rdf = 'rdflib.tools.csv2rdf:main'
+rdf2dot = 'rdflib.tools.rdf2dot:main'
+rdfs2dot = 'rdflib.tools.rdfs2dot:main'
+rdfgraphisomorphism = 'rdflib.tools.graphisomorphism:main'
+
+[tool.poetry.dependencies]
+python = "^3.7"
+isodate = "^0.6.1"
+pyparsing = "^3.0.9"
+#setuptools  # not needed anymore?
+importlib_metadata = {version = "^4.2.0", python = ">=3.7,<3.8"}
+berkeleydb = {version = "^18.1.5", optional = true}
+networkx = [
+    {version = "^2.6.2", python = ">3.7,<3.8", optional = true},
+    {version = "^2.8.8", python = "^3.8", optional = true},
+]
+
+[tool.poetry.group.dev.dependencies]
+html5lib = "^1.1"
+pytest = "^7.1.3"
+pytest-cov = "^4.0.0"
+myst-parser = "^0.18.0"
+sphinx = ">4.0.0"
+sphinxcontrib-apidoc = "^0.3.0"
+sphinxcontrib-kroki = "*"
+sphinx-autodoc-typehints = "*"
+black = "22.12.0"
+flake8 = [
+    {version = "^5.0.0", python = ">=3.7,<3.8"},
+    {version = "^6.0.0", python = ">=3.8.1"},
+]
+isort = "^5.10.0"
+mypy = "0.991"
+
+[tool.poetry.extras]
+berkeleydb = ["berkeleydb"]
+networkx = ["networkx"]
```

Comment:
```suggestion
html = ["html5lib"]
```

Reply:
I think this suggestion is resolved with my latest changes?